### DISCUSSION TITLE
Sunbird inQuiry setup

### DISCUSSION THREAD

**Discussion by vivek-m-gowda** [Sunbird-inQuiry/Community#61] [Q&A] [answered]
Hello everyone, I'm attempting to locally install Sunbird inQuiry using the URL at https://inquiry.sunbird.org/use/developer-installation. I can see from this point onward that there are three separate apps available: Question & Question Set Service [knowledge-platform], Question Set Editor [sunbird-collection-editor], and QuML Player Contribution [sunbird-quml-player]. Could you please let me know which branch has been most recently committed for all three github repository? or which branch I should check out from.

**Comment by rajnishdargan** [comment, marked as answer]
Hi @vivek-m-gowda Latest stable branch is release-5.4.0, you can refer the [release-notes](https://inquiry.sunbird.org/use/release-notes/inquiry-release-v5.4.0-latest) for all the release information.

**Reply by vivek-m-gowda** [reply]
Thanks @rajnishdargan, marking it as answer
